Sources/SwiftSyntax/generated/TokenKind.swift

```diff
@@ -724,51 +724,71 @@ public enum RawTokenKind: UInt8, Equatable, Hashable {
   }
 }
 
+private extension RawTokenKind {
+  var defaultTextString: String? {
+    guard let defaultText else {
+      return nil
+    }
+    return String(syntaxText: defaultText)
+  }
+}
+
 extension TokenKind {
   /// If the `rawKind` has a `defaultText`, `text` can be empty.
   @_spi(RawSyntax)
   public static func fromRaw(kind rawKind: RawTokenKind, text: String) -> TokenKind {
     switch rawKind {
     case .arrow:
-      precondition(text.isEmpty || rawKind.defaultText.map(String.init) == text)
+      let isNotDefaultText: Bool = rawKind.defaultTextString == text
+      precondition(text.isEmpty || isNotDefaultText)
       return .arrow
     case .atSign:
-      precondition(text.isEmpty || rawKind.defaultText.map(String.init) == text)
+      let isNotDefaultText: Bool = rawKind.defaultTextString == text
```

Comment:
I have tried different configurations, this one is the fastest

Reply:
Thank you, I forgot about expression evaluation within the `precondition` autoclosure argument.

To my surprise, the only configuration that worked, which I did not try before your review, is this:
```swift
precondition(((text.isEmpty as Bool) || ((rawKind.defaultTextString == text) as Bool)) as Bool)
```

Bool for all 🎆